src/commands/env/var/set.ts

```diff
@@ -118,42 +68,33 @@ export default class ConfigSet extends Command {
     }
 
     if (flags.environment) {
-      cli.warn(messages.getMessage('flags.environment.deprecation'));
+      this.warn(messages.getMessage('flags.environment.deprecation'));
     }
 
     const appName = await resolveAppNameForEnvironment(targetCompute);
 
-    if (flags.json) {
-      try {
-        const configPairs = this.parseKeyValuePairs(argv);
-
-        await this.client.patch(`/apps/${appName}/config-vars`, {
-          data: configPairs,
-        });
-
-        cli.styledJSON({
-          status: 0,
-          result: null,
-          warnings: [],
-        });
-        return;
-      } catch (err: any) {
-        this.parseErrors(err, targetCompute);
-      }
-    } else {
-      const configPairs = this.parseKeyValuePairs(argv);
+    const configPairs = this.parseKeyValuePairs(argv);
 
-      cli.action.start(
-        `Setting ${Object.keys(configPairs)
-          .map((key) => herokuColor.configVar(key))
-          .join(', ')} and restarting ${herokuColor.app(targetCompute)}`
-      );
+    cli.action.start(
+      `Setting ${Object.keys(configPairs)
+        .map((key) => herokuColor.configVar(key))
+        .join(', ')} and restarting ${herokuColor.app(targetCompute)}`
+    );
 
+    try {
       await this.client.patch(`/apps/${appName}/config-vars`, {
         data: configPairs,
       });
 
       cli.action.stop();
+
+      return [];
```

Comment:
I won't hold up this PR for this but you should return _something_.
For these types of commands we often return the thing/object that was set.